Makefile

```diff
@@ -18,7 +18,7 @@
 UNAME:=$(shell uname -s | tr [A-Z] [a-z])
 BRANCH:=$(shell git branch --show-current)
 VENV:=venv.$(UNAME).$(BRANCH)
-PYTHON:=python
+PYTHON?=python
```

Comment:
I'm not opposed to changing this, but the other assignments, particularly `BRANCH` should probably work correspondingly.

Reply:
Done